### ISSUE TITLE
floating arrow head

### ISSUE THREAD

**Issue by SepidehAlassi** [vasturiano/3d-force-graph#320]
**Describe the bug**
I am visualizing my graph in an Angular 9 app, using the 3D-force-graph npm package. When I change the links to be directional, I get a floating arrowhead right in the middle of the canvas. 
This floating arrowhead only appears when I use the npm package. The same data is visualized correctly with the JS version.
**To Reproduce**
1. Go to [ontoVis repository](https://github.com/SepidehAlassi/ontoVis)
2. Run 'npm i' and 'ng s'

**Expected behavior**
No floating arrowhead.
**Screenshots**
<img width="787" alt="Screenshot 2020-05-07 10 51 27" src="https://user-images.githubusercontent.com/8567642/81274566-c24f9400-9050-11ea-8519-8af986399a42.png">
**Desktop (please complete the following information):**
 - OS: macOS Mojave
 - Browser: chrome and safari

**Comment by vasturiano** [comment]
@SepidehAlassi thanks for reaching out.

It sounds like you could have a detached link. Such a link would be rendered in the center or the canvas and if there are arrows associated with every link, that would explain the floating arrow.

If the issue persists, would you mind making a minimal example on [codesandbox](https://codesandbox.io/) that reproduces the issue?

**Comment by SepidehAlassi** [comment]
@vasturiano Thank you very much for your quick reply.

I checked my data for a detached link, but I realized that a circular link that has the same source and target causes the problem. Is there any option to enable circular links? 

I understand that considering the force-engine and calculation of attractive and repulsive forces this might not be possible. A workaround would be creating a dummy node for the target to make it different from the source node.
